src/Components/CustomComponent/ButtonComponent.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+export default function Button(props){
+    function clickHandler(){
+        alert(`You Clicked on ${props.name}`)
+    }
+    return(
+
+
+        <button onClick={clickHandler}>{props.name}</button>
+    )
+}
```

Comment:
If it is possible for you, I would like to suggest using destructuring instead of referencing props directly. This can make the code cleaner and neater. Please note that this is optional and based on personal preference.

⚠️ OPTIONAL